packages/realm-server/tests/cards/person.json

```diff
@@ -0,0 +1,14 @@
+{
+  "data": {
+    "type": "card",
+    "attributes": {
+      "firstName": "Mango 2"
+    },
+    "meta": {
+      "adoptsFrom": {
+        "module": "./person.gts",
+        "name": "Person"
+      }
+    }
+  }
+}
```

Comment:
hanging file?

Reply:
added this so we have two files with same name before the extension. `person.gts` and `person.json`. we're doing a request for `person` in the tests